lib/pxcontext.js

```diff
@@ -12,6 +13,7 @@ class PxContext {
         this.score = 0;
         this.ip = PxContext.extractIP(config, request);
         this.headers = request.headers;
+        this.filteredHeaders = pxUtil.filterSensitiveHeaders(request.headers);
```

Comment:
I think it's better to just save the filtered headers as `this.headers`.
Then you can remove the `pxConfig.SENSITIVE_HEADERS.indexOf(header) === -1` condition from [here](https://github.com/PerimeterX/perimeterx-node-core/pull/32/files#diff-c3e62b60d3b0f965ec22a2222b7e0746R103), and every other call for `pxUtil.filterSensitiveHeaders`